client/a8c-for-agencies/sections/sites/features/a4a/site-details.tsx

```diff
@@ -12,25 +15,29 @@ export default function SiteDetails( { site }: any ) {
 	const translate = useTranslate();
 	const dispatch = useDispatch();
 	/* eslint-disable-next-line */
-	const { a4a_site_id: siteId, a4a_site_tags: initialTags } = site;
+	const { a4a_site_id: siteId, a4a_site_notes: initialNotes, a4a_site_tags: initialTags } = site;
 
 	const [ tags, setTags ] = useState(
 		initialTags ? initialTags.map( ( tag: SiteTagType ) => tag.label ) : []
 	);
+
+	const [ notes, setNotes ] = useState( initialNotes ?? [] );
```

Comment:
Same feedback here about notes as with tags and propagating changes up and down back to this component.